src/addons.c

```diff
@@ -60,7 +60,7 @@ static void inbox_received_callback(DictionaryIterator *iterator, void *context)
     dict_write_cstring(&items_iter, (int) t->key, (char*) t->value->cstring);
     t = dict_read_next(iterator);
   }
-  int items_size = dict_write_end(&items_iter);
```

Comment:
Was this ever used? Unused defined variables are warning and by default warnings are now errors.